Gemfile

```diff
@@ -23,11 +24,11 @@ gem 'awesome_nested_set', '~> 3.0.0.rc.1'
 gem 'cancan', '~> 1.6.10'
 gem 'ffaker', '~> 1.16'
 gem 'highline', '2.0.3' # Necessary for the install generator
-gem 'json', '>= 1.7.7'
-gem 'money', '5.1.1'
+gem 'json'
+gem 'money', '< 6.1.0'
```

Comment:
this update needs a testing note to be manually checked. currencies and potential price rounding issues?

Reply:
There are lots of problems if we bump to `6.1.0` or higher. It's a cap on updates, rather than an update. It's tracked in this issue: https://github.com/openfoodfoundation/openfoodnetwork/issues/6333#issuecomment-723059261

Reply:
it's actually downgrading it!?
![image](https://user-images.githubusercontent.com/1640378/99080816-bc1b1100-25b9-11eb-9aff-9cb49a1c0531.png)

maybe we can keep using 5.1.1?

re < 6, I would add a short comment to the Gemfile to explain that lock, so that the next dev looking at this will not have to look around to find out.

Reply:
It looks like specifying `5.1.1` now conflicts with other dependencies, for `spree_i18n`. @Matt-Yorkley suggested in Slack to pull that code into the codebase and remove the gem. I think keeping the `< 6.1.0`, with a comment, and carefully testing features to do with money/rounding is the way forward for now (and maybe taking a look at the changelog for the money gem from `5.0` to `5.1.1`.